ckanext/dcatapit/interfaces.py

```diff
@@ -306,7 +306,7 @@ def get_license_for_dcat(license_type):
     l = License.get(license_type or License.DEFAULT_LICENSE)
     if not l or not l.license_type:
         l = License.get(License.DEFAULT_LICENSE)
-    names = l.get_names()
+    names = dict((k['lang'], k['name']) for k in l.get_names())
```

Comment:
In EN and DE we have **Unknown IPR** instead of **Unknown License** and **Unbekannte Lizenz** respectively. Are we using the right default license from the vocabulary?

Reply:
Yes, because `DEFAULT_LICENSE` is defined as id, not label, see https://github.com/geosolutions-it/ckanext-dcatapit/blob/master/ckanext/dcatapit/model/license.py#L79

Reply:
Maybe are we using the wrong ID since in the vucabulary we have:

https://w3id.org/italia/controlled-vocabulary/licences/C_Unknown

that contains the right translations.